hack/test-go.sh

```diff
@@ -45,6 +45,8 @@ KUBE_TIMEOUT=${KUBE_TIMEOUT:--timeout 120s}
 KUBE_COVER=${KUBE_COVER:-} # set to nonempty string to enable coverage collection
 KUBE_COVERMODE=${KUBE_COVERMODE:-atomic}
 KUBE_RACE=${KUBE_RACE:-}   # use KUBE_RACE="-race" to enable race testing
+# How many processes to use when running tests.
+KUBE_PROCS=${KUBE_PROCS:-$(grep -c '^processor' /proc/cpuinfo)}
```

Comment:
Looks flaky to me.

Will probably annoy people building it on MacOS and I know of quite a few of them.

Isn't this actually the default in Go? Can't it just take a parameter to do the auto-detection by itself?